scheduler/README.md

```diff
@@ -4,19 +4,16 @@
 
 ### Running the application
 
-There are two ways to start the NodeJs server:
-
-- the preferred way is running `docker-compose docker-compose -f ../docker-compose.yml -f ../docker-compose.ci.yml up adapter-service up scheduler-service`
-- if you don't want to use docker-compose you can also run `npm start` in the project root directory. Make sure all dependencies are downloaded by running `npm install` beforehand.
-
-The server should now be running on http://localhost:8080
+Since the scheduler performs an initial sync with the adapter service, it is currently only available to start in in the docker environment:
+Run `docker-compose -f ../docker-compose.yml up scheduler adapter`. 
 
 ### Testing
 
 Run the unit tests with `npm test`. Jest is used as unit testing framework.
 
 For integration testing run `docker-compose -f ../docker-compose.yml -f ../docker-compose.ci.yml up scheduler-service scheduler-service-it`.
+To run integration tests outside of the docker environment run the `it_local.sh` script.
```

Comment:
I think we should discuss about this script. Is it something we want everywhere? Do we better provide env files instead?

Reply:
Yeah, I think so too. I added this script for local testing, and I included it into the pr just in case somebody else may need it, too. I will create a discussion issue so we can find a common approach for all services. Maybe we can discuss it in the next dev meeting as well.